main.go

```diff
@@ -94,8 +103,16 @@ func main() {
 		dief("looking for test data: %v\n", err)
 	}
 	for _, name := range remove {
-		if err := os.RemoveAll(name); err != nil {
-			dief("removing %v: %v\n", name, err)
+		err := os.RemoveAll(name);
+		if err != nil {
+			failedToRemove[name] = err
+		}
+	}
+	
+	if len(failedToRemove) > 0 {
+		fmt.Fprintf(os.Stderr, "\n# Failed to remove below item, please remove manually\n")
+		for name, err := range failedToRemove {
```

Comment:
It won't rarely happen; it will happen every time. The git program sets a permission bit on the `.git` folder. We need to fix it using the workaround I suggested to say this is supported on Windows.